src/sinks/mezmo.rs

```diff
@@ -24,10 +24,40 @@ use crate::{
 
 const PATH: &str = "/logs/ingest";
 
-/// Configuration for the `logdna` sink.
+/// Configuration for the `LogDNA` source.
```

Comment:
```suggestion
/// Configuration for the `logdna` sink.
```

Shouldn't it be this?